react/features/conference/components/web/Subject.js

```diff
@@ -2,11 +2,11 @@
 
 import React, { Component } from 'react';
 
+import TimeElapsed from '../../../../features/speaker-stats/components/TimeElapsed';
 import { getConferenceName } from '../../../base/conference/functions';
-import { getParticipantCount } from '../../../base/participants/functions';
+import { getParticipantCount, getDominantSpeaker } from '../../../base/participants/functions';
```

Comment:
I think it will also work like that:
```suggestion
import { getParticipantCount, getDominantSpeaker } from '../../../base/participants';
```